### DISCUSSION TITLE
Roles rate limit, what is it?

### DISCUSSION THREAD

**Discussion by tassoman** [misskey-dev/misskey#12326] [Q&A]
## What is Rate Limit? 

![Screenshot 2023-11-13 at 23-53-32 BETA Misskey Social](https://github.com/misskey-dev/misskey/assets/92727/14d3910e-9aae-40b2-9dff-19a025e5925c)

## And what is priority, while overriding "template role" basic values?

![Screenshot 2023-11-13 at 23-54-26 BETA Misskey Social](https://github.com/misskey-dev/misskey/assets/92727/1a527931-8c47-4a35-a4db-550749df9f27)

**Comment by spirillen** [comment]
Good question...

**Reply by tassoman** [reply]
Got answered in this issue https://github.com/misskey-dev/misskey/issues/13873
